src/ScillaContractDeployer.ts

```diff
@@ -49,6 +60,52 @@ export class ScillaContract extends Contract {
   [key: string]: ContractFunction | any;
 }
 
+function handleParam(param:Field, arg:any, init_call:boolean = true):Value{
```

Comment:
IMO it would be cleaner to just have 2 separate functions here - The logic between `init_call = true` and `init_call = false` seem to be dissimilar enough that sticking them together doesn't achieve much.